modules/Microsoft.ManagedIdentity/userAssignedIdentities/deploy.bicep

```diff
@@ -70,6 +70,9 @@ output resourceId string = userMsi.id
 @description('The principal ID of the user assigned identity.')
```

Comment:
```suggestion
@description('The principal ID (object ID) of the user assigned identity.')
```

@davidkarlsen @AlexanderSehr 

I would like to suggest that we rename the output descriptions to something that makes it easier to distinguish what each ID is.

Reply:
@ahmadabdalla agree, that was also discussed in issue #3113. Outside of this PR I'd suggest to open an issue to align all module outputs supporting managed identities accordingly

Reply:
adding docs, yes, changing var names no. They should be as in the ARM/bicep API to avoid confusion.

Reply:
opened issue #3139